src/consts.ts

```diff
@@ -6,7 +6,7 @@ export const ZOOM_NON_DESKTOP_DEFAULT_STEP = 0.04 // transform scale
 export const ZOOM_MIN_DEFAULT = 0.3
 export const ZOOM_MAX_DEFAULT = 5
 export const ZOOM_SPEED_DEFAULT = 1
-export const ZOOM_PRECISION = 100.0;
+export const ZOOM_PRECISION = 10000.0
```

Comment:
good point, thank you. Do you find out that `100.0` was leading to zoom out instead of zoom in on some cases? or it was something different?

Reply:
@sasza2 it was something different. With great images to zoom out until they're very smaller than the original one, it's just an approximation error that becomes visible. You can see what I mean by trying this commit locally on your computer replacing the zoom precision back to 100.
Honestly, with the image I chose 1000.0 would be fair too, but just in case, I've set this to 10k

Reply:
thanks, good eye